.github/workflows/nightly-wheels.yml

```
@@ -0,0 +1,41 @@
+name: Upload nightly wheels
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+jobs:
+  cron:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

we need `fetch-depth: 0` to reliably determine the correct version:
```suggestion
      - uses: actions/checkout@v3
        with:
          fetch-depth: 0
```